manager/middlewares/rbac.go

```diff
@@ -56,9 +67,19 @@ func RBAC() gin.HandlerFunc {
 	}
 
 	return func(c *gin.Context) {
-		if !isAccessGranted(c) {
-			c.AbortWithStatusJSON(http.StatusUnauthorized,
-				utils.ErrorResponse{Error: "You don't have access to this application"})
+		grantedPerms := isAccessGranted(c)
+
+		switch c.Request.Method {
+		case "GET", "POST":
```

Comment:
I suppose this works, but wouldn't it be better if we performed read check for every endpoint, and write check manually in the delete_system endpoint ? By storing the perms in the context, like we do  with the account id.

Reply:
Yes, but this is easier solution. Just because we don't to go directly into the handler's code. I would keep it like this at least for now. I don't think we will need to modify it in the future. If so we can do it once it will be neccassarry.